src/DetailsView/components/adhoc-tab-stops-test-view.tsx

```diff
@@ -94,6 +94,27 @@ export const AdhocTabStopsTestView = NamedFC<AdhocTabStopsTestViewProps>(
                         </li>
                     </ol>
                 </li>
+                {props.featureFlagStoreData != null &&
+                props.featureFlagStoreData[FeatureFlags.tabStopsAutomation] ? (
+                    <li>
+                        Record your results for each requirement:
+                        <ol>
+                            <li>
+                                If you find any failures, select <Markup.Term>Fail</Markup.Term>,
+                                then add then as failure instances.
```

Comment:
There seems to be a typo

```suggestion
                                then add them as failure instances.
```